pages.tr/common/tar.md

```diff
@@ -0,0 +1,37 @@
+# tar
+
+> Arşivleme aracı.
+> Genellikle gzip veya bzip2 gibi bir sıkıştırma yöntemiyle birleştirilir.
+> Daha fazla bilgi için: <https://www.gnu.org/software/tar>.
```

Comment:
```suggestion
> Daha fazla bilgi: <https://www.gnu.org/software/tar>.
```
I don't think we need for (için) here in order to better align with our other pages. [Reference](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py)